### PR TITLE
Update header of README

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
Consistency!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,48 @@
...
+</a>
+<a href="https://github.com/jellyfin/jellyfin-apiclient-java/commits/master.atom">
+<img alt="Master Commits RSS Feed" src="https://img.shields.io/badge/rss-commits-ffa500?logo=rss" />
+</a>
 </p>
 
 ---
 
-This library allows Android clients to easily access the Jellyfin API. It is built with Volley, OkHttp, Boon, and Robolectric. The dependencies are modular and can easily be swapped out with alternate implementations when desired.
+This library allows Java and Android applications to easily access the Jellyfin API. It is built with Volley, OkHttp, Boon, and Robolectric. The dependencies are modular and can easily be swapped out with alternate implementations when desired.
```
Comment (thornbill) [inline README.md:45]: This list of dependencies seem to be completely inaccurate (we only use Volley out of that list). Should we just remove that part?

Comment (nielsvanvelzen) [reply README.md:45]: I was going to update that part after refactoring some code. I'm looking into replacing volley too.

Comment (thornbill) [reply README.md:45]: I have a local implementation that uses OkHttp instead but the issue I encountered was ATV needs refactored to allow http requests to run on a different thread.

Comment (nielsvanvelzen) [reply README.md:45]: I'll take that in mind. When 0.7.0 gets released I will update the ATV app too

**Pull request merged**